test/libgit2-online.jl

```diff
@@ -21,12 +21,17 @@ mktempdir() do dir
                 finalize(repo)
             end
         #end
+        try
         #@testset "with incorrect url" begin
             repo_path = joinpath(dir, "Example2")
             # credential are required because github try authenticate on uknown repo
-            x = Nullable(LibGit2.UserPasswordCredentials("X","X"))
-            @test_throws LibGit2.Error.GitError LibGit2.clone(https_prefix*repo_url*randstring(10), repo_path, payload=x)
+            cred = LibGit2.UserPasswordCredentials("","") # empty credentials cause authentication error
+            LibGit2.clone(https_prefix*repo_url*randstring(10), repo_path, payload=Nullable(cred))
```

Comment:
is expected to always fail? if so it should throw a different type of exception at the end of the try so it doesn't silently succeed